app/views/tasks/index.html.erb

```diff
@@ -14,8 +14,15 @@
       Update Task
```

Comment:
You did a really nice job here making sure that you markup is formatted and indented well